common/include/pcl/conversions.h

```diff
@@ -205,10 +205,10 @@ namespace pcl
     else
     {
       // If not, memcpy each group of contiguous fields separately
-      for (std::uint32_t row = 0; row < msg.height; ++row)
+      for (std::size_t row = 0; row < msg.height; ++row)
```

Comment:
@haritha-j Does one of your PR address this change?

Reply:
I issued a pull request on Monday related to the loading and saving of files. The one I have put in just now relates to conversion to templated types. I have used the same fork for the pull request though. If this is an issue I can create another fork an put the request in again.

Reply:
@kunaltyagi in #4199 i changed `height` of `pointCloud2` to `index_t`. I have also replaced some instances of `std::size_t` with `index_t`. I basically updated the instances where `sign-compare` warnings came up. This file doesn't seem to be touched there.